tests/foreman/virtwho/test_cli_virtwho.py

```diff
@@ -89,17 +89,23 @@ def test_positive_deploy_configure_by_id(self):
         self.assertEquals(
             VirtWhoConfig.info({'id': vhd['id']})['general-information']['status'], 'OK')
         hosts = [
-            (hypervisor_name, 'product_id={}'.format(self.vdc_physical)),
-            (guest_name, 'type=STACK_DERIVED')]
+            (hypervisor_name, 'product_id={} and type=NORMAL'.format(self.vdc_physical)),
+            (guest_name, 'product_id={} and type=STACK_DERIVED'.format(self.vdc_physical))]
         for hostname, sku in hosts:
             host = Host.list({'search': hostname})[0]
-            vdc = Subscription.list({
+            subscriptions = Subscription.list({
                 'organization': DEFAULT_ORG,
                 'search': sku,
-            })[0]
+            })
+            vdc_id = subscriptions[0]['id']
+            if 'type=STACK_DERIVED' in sku:
+                for item in subscriptions:
+                    if hypervisor_name in item['type']:
```

Comment:
Can you confirm, that this condition is correct? That `hypervisor_name` will be in `item['type']`? 

I noticed that in another file, condition is `if hypervisor_name.lower() in item['hypervisor']['name']`.

Reply:
Yes,  the subscription list result is difference between cli and api.
**to cli, can check the hypervisor_name by type:**
```
    {
        "account": "xxx",
        "consumed": "0",
        "contract": "xxx",
        "end-date": "2020/05/24 03:59:59",
        "id": "39",
        "name": "Red Hat Enterprise Linux for Virtual Datacenters, Premium",
        "quantity": "Unlimited",
        "support": "Premium",
        "type": "Guests of virt-who-win-virt-who-ci-1",
        "uuid": "8ac213ba6cf11645016cfc6db3710d63"
    }
```
**to api**, I have to check hypervisor_name by ['hypervisor']['name'] :
```
    {
        "id": 24,
        "cp_id": "xxx",
        "subscription_id": 7,
        "name": "Red Hat Enterprise Linux for Virtual Datacenters, Premium",
        "start_date": "2019-05-24 04:00:00 UTC",
        "end_date": "2020-05-24 03:59:59 UTC",
        "available": -1,
        "quantity": -1,
        "consumed": 0,
        "account_number": xxx,
        "contract_number": xxx,
        "support_level": "Premium",
        "product_id": "RH00049",
        "sockets": "2",
        "cores": null,
        "ram": null,
        "instance_multiplier": 1,
        "stacking_id": null,
        "multi_entitlement": null,
        "type": "STACK_DERIVED",
        "product_name": "Red Hat Enterprise Linux for Virtual Datacenters, Premium",
        "unmapped_guest": false,
        "virt_only": true,
        "virt_who": false,
        "upstream": true,
        "upstream_pool_id": null,
        "hypervisor": {
            "id": 67,
            "name": "virt-who-win-virt-who-ci-1"
        }
    }
```